api/JavascriptBindings.md

```diff
@@ -131,23 +133,53 @@ Currently this function binds only methods of an object. Example:
 	myobject.someMethod();
 ```
 
-Currently when binding object only methods are binded, I decided not to bind properties of the object, as they would be binded by copying value and this might be confusing, as accessing object's property from javascript might give a different value during runtime then the real value when getting the property from python runtime. Only object's methods and functions can be binded by reference. Still you can bind object's properties if you like, you can find useful method IsValueAllowed() to check which properties can be binded, of course doing it this way will not allow you to access properties through "window.myobject.property", you can only bind to the "window" object so you should imitate some kind of namespace, so that accessing property would be through "window.myobject_property" or "myobject_property" as window prefix is always optional. Use dir() function to list object's properties. Example code:
+By default when binding object only methods are bound unless `allow_properties` is explicitely set. This was done as properties can currently ounly be bound by copying value during the binding process. It is expected this could be confusing as accessing object's property from javascript might give a different value during runtime then the real value when getting the property from python runtime if either end has been changed in the mean time. Only object's methods and functions can be bound by reference currently.  
```

Comment:
well spotted. I've figured out how to turn spell checker back on in pycharm :-) will fix